src/components/AvatarWithIndicator.js

```diff
@@ -61,11 +59,17 @@ const AvatarWithIndicator = (props) => {
         isLarge ? styles.statusIndicatorLarge : styles.statusIndicator,
     ];
 
-    const hasPolicyMemberError = _.some(props.policiesMemberList, policyMembers => Policy.hasPolicyMemberError(policyMembers));
-    const hasCustomUnitsError = _.some(props.policies, policy => Policy.hasCustomUnitsError(policy));
-    const hasPaymentMethodError = PaymentMethods.hasPaymentMethodError(props.bankAccountList, props.cardList);
-    const hasWalletError = !_.isEmpty(props.userWallet.errors);
-    const shouldShowIndicator = hasPolicyMemberError || hasPaymentMethodError || hasWalletError || hasCustomUnitsError;
+    // All of the error-checking methods are put into an array. This is so that using _.some() will return
+    // early as soon as the first error is returned. This makes the error checking very efficient since
+    // we only care if a single error exists anywhere.
+    const errorCheckingMethods = [
+        () => !_.isEmpty(props.userWallet.errors),
+        () => PaymentMethods.hasPaymentMethodError(props.bankAccountList, props.cardList),
+        () => _.some(props.policies, PolicyUtils.hasPolicyError),
```

Comment:
This line is most likely causing https://expensify.slack.com/archives/C01GTK53T8Q/p1661773256971749

Seems like `props.policies` is null